spec/database/database_schema_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe "Database" do
+  describe "foreign key constraints" do
+    let(:query) do
+      <<-SQL
+        SELECT
+          constraint_name, table_name
+        FROM
+          information_schema.table_constraints
+        WHERE
+          constraint_type = 'FOREIGN KEY'
+      SQL
+    end
+
+    it "should not exist" do
+      message = ""
+      ActiveRecord::Base.connection.select_all(query).each do |fk|
+        message << "Foreign key constraint #{fk["constraint_name"]} on table #{fk["table_name"]} should be removed\n"
+      end
+      raise message unless message.empty?
```

Comment:
Doesn't really matter but might as well go through RSpec's failure API in case that matters someday (use `fail("message")` instead of `raise`)